turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "tasks": {
+    "compile": {
```

Comment:
can we name this something like `compileTypescript` or something?

(Perhaps we should also rename the other ones to indicate that they refer to the Rust side)